package.json

```diff
@@ -195,6 +195,7 @@
 		"format": "prettier . --check",
 		"format:fix": "prettier . --write",
 		"test": "vscode-test",
+		"test:dev": "xvfb-run -a npm run test",
```

Comment:
This will fail on systems that do not have `xvfb-run` installed.  We should probably abstract this into a script that check the OS as well as if that specific binary is installed.  If the OS is Linux, and the binary isn't installed, we should throw an error.  If the OS isn't Linux, we should run `npm run test` directly.

Additionally, this script should probably be renamed to `test:web-ide` or something that denote's its intended use a bit better.

Reply:
I have updated the title to better reflect the actual intent. What I want to achieve is running npm run test:ci locally.